charts/cassandra/templates/deployment.yaml

```diff
@@ -134,6 +134,9 @@ spec:
       protocol: TCP
       port: 7000
       targetPort: 7000
+    - name: exporter
```

Comment:
I would define the port in the pod / container spec instead of the service. The prometheus scrape won't use the clusterIP service anyway, it scrapes pods directly.

Reply:
Sure thing! Thanks for the feedback!